pkg/services/sqlstore/org_test.go

```diff
@@ -93,6 +93,44 @@ func TestAccountDataAccess(t *testing.T) {
 				So(q1.Result[0].OrgId, ShouldEqual, q2.Result[0].OrgId)
 				So(q1.Result[0].Role, ShouldEqual, "Viewer")
 			})
+
+		})
+
+		Convey("Given single org and 2 users inserted", func() {
+			setting.AutoAssignOrg = true
+			setting.AutoAssignOrgId = 1
+			setting.AutoAssignOrgRole = "Viewer"
```

Comment:
Changes to state need to be cleaned up after the test is over, otherwise other tests get affected, which is a no-no:
```suggestion
			origAutoAssignOrg := setting.AutoAssignOrg
			origAutoAssignOrgId := setting.AutoAssignOrgId
			origAutoAssignOrgRole := setting.AutoAssignOrgRole
			t.Cleanup(func() {
				setting.AutoAssignOrg = origAutoAssignOrg
				setting.AutoAssignOrgId = origAutoAssignOrgId
				setting.AutoAssignOrgRole = origAutoAssignOrgRole
			})
			setting.AutoAssignOrg = true
			setting.AutoAssignOrgId = 1
			setting.AutoAssignOrgRole = "Viewer"
```